### DISCUSSION TITLE
SMTP connect() failed. https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting

### DISCUSSION THREAD

**Discussion by LeDxniel** [PHPMailer/PHPMailer#3029] [Q&A]
I want to create a simple contact form. However, this message always comes up when sending. Does anyone have an idea what the problem is?
Here is my code
https://pastebin.com/2EbfUtd4

**Comment by Synchro** [comment]
What happened when you followed the advice in the documentation linked from the error message? What does your debug output show when you set `$mail->SMTPDebug = 3`, as the docs suggest?
